brut.apktool/apktool-lib/src/main/java/brut/androlib/res/data/ResConfigFlags.java

```diff
@@ -144,12 +144,7 @@ private String generateQualifiers() {
             ret.append("-mcc").append(String.format("%03d", mcc));
             if (mnc != MNC_ZERO) {
                 if (mnc != 0) {
-                    ret.append("-mnc");
-                    if (mnc > 0 && mnc < 10) {
-                        ret.append(String.format("%02d", mnc));
-                    } else {
-                        ret.append(String.format("%03d", mnc));
-                    }
+                    ret.append(String.format("-mnc%03d", mnc));
```

Comment:
This shouldn't be changed. Unless you know of a different way to do it :)

mnc values between 1 and 9 require only two digits (0 padded). Your change makes it `001` which will break the unit-tests because it is expecting `01` like AOSP requires.